src/benchmark/ips.cr

```diff
@@ -60,18 +60,17 @@ module Benchmark
         @items.each do |item|
           GC.collect
 
-          before = Time.now
-          target = Time.now + @warmup_time
-          count = 0
-
-          while Time.now < target
-            item.call
-            count += 1
+          elapsed = Time.measure do
+            target = Time.monotonic + @warmup_time
+            count = 0
```

Comment:
`count` needs to be defined outside the block.

Reply:
Pushed too early.